pytest.ini

```diff
@@ -0,0 +1,8 @@
+[pytest]
+filterwarnings =
+    ignore:.*'soft_unicode' has been renamed to 'soft_str'*:DeprecationWarning
+    ignore:unclosed file .*:ResourceWarning
+env_files =
+    test.env
+testpaths =
+    tests/functional
```

Comment:
Last Week I Learned (LWIL):
- [By default `pytest-dotenv` will not override any variables already defined in the process' environment](https://github.com/quiqua/pytest-dotenv/blob/866c90d9475bc469a0e9fc848868af0139658786/README.md#overriding-existing-values)

### Example
Let's say I have an environment variable defined locally:
`DBT_DATABASE_HOST=localhost`

And `test.env` contains:
`DBT_DATABASE_HOST=somethingelse`

By default, pytest-dotenv will keep the former and ignore the latter. Scroll down to see how to configure the opposite behavior.

### Override default behavior
So it depending on how we want pytest + pytest-dotenv to behave, we either will want to include the following (or not!):
```
env_override_existing_values = 1
```

Reply:
A great LWIL!

This makes sense to me, insofar as I might have a `test.env`, but then also want to do:
```
export DBT_DATABASE_HOST=localhost
python3 -m pytest ...
```
I would indeed expect the env var I've just supplied to override the one in `test.env`